libs/ui/src/pure/ProductLogo/index.tsx

```diff
@@ -270,7 +282,8 @@ export const ProductLogo = ({
   external = false,
 }: LogoProps) => {
   const themeMode = useTheme()
-  const logoForTheme = LOGOS[variant][customThemeMode || (themeMode.darkMode ? 'dark' : 'light')]
+  const selectedTheme = customThemeMode || (themeMode.darkMode ? 'dark' : 'light')
+  const logoForTheme = LOGOS[variant][selectedTheme] || LOGOS[variant]['light'] // Fallback to light theme if selected theme is not available
```

Comment:
`const logoForTheme = LOGOS[variant][selectedTheme]` is enough since `selectedTheme` always has a value

Reply:
If I do that then `'logoForTheme' is possibly 'undefined'.` and we need to modify things with `logoForTheme.logoIconOnly` to `logoForTheme?.logoIconOnly`